docker-compose.yml

```diff
@@ -35,16 +41,6 @@ services:
     expose:
       - '4444'
     shm_size: 2g
-  node:
-    image: node:22.11.0
-    user: "node"
-    working_dir: /home/node/app
-    environment:
-      - NODE_ENV=${NODE_ENV}
-      - SHELL=bash
-    volumes:
-      - ./:/home/node/app
-    command: "sleep infinity"
```

Comment:
We are removing the 'node' container, but the README.md still talks about `docker-compose exec -u node node npm install`.
Are we now relying on npm in the web container? If that is sufficient, why did we need this separate container in the past?

Reply:
I agree we should update README to reflect the changes and how oe_theme is supposed to be built from now on.

Reply:
I think there are two changes:
1. In the general setup we no longer need to call the npm steps, they are part of `drupal:site-setup` which is called with composer install and composer update.
2. We could still mention the steps for front-end development.